libs/core/src/lib/object-marker/object-marker.component.scss

```diff
@@ -0,0 +1,2 @@
+@import '~fundamental-styles/dist/icon';
```

Comment:
import of Icons is not needed there, because it's imported already in `angular.json` and will be imported that way by end-users. It's just to reduce size of output file.